crashDataAnalysisTools/plot_and_save.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pandas as pd
+from matplotlib.backends.backend_pdf import PdfPages
+
+
+def PlotXVsY(df, colname_x, colname_y):
+    """
+    scatter plot of attribute x against y
+    Parameters:
+    @df {dataframe} datasource
+    @colname_x {string} column name for attribute x
+    @colname_y {string} column name for attribute y
+    Return:
+    the produced figure
+    """
+    plt.scatter(df[colname_x], df[colname_y])
+    plt.xlabel(colname_x)
+    plt.ylabel(colname_y)
+    plt.show()
+    return plt.figure()
+
+
+def PlotModelCDF(rmodel, lowerb, upperb, modelname):
+    """
+    plot of model CDF
+    Parameters:
+    @model a regression model, e.g. Logit model or Probit model
+    @lowerb {double} the lower bound of the dataset
+    @upperb {double} the upper bound of the dataset
+    @modelname {string} the name of the model, e.g. Probit or Logit
+    Return:
+    the produced figure
+    """
+    fig = plt.figure(figsize=(12, 8))
+    ax = fig.add_subplot(111)
+    support = np.linspace(-6, 6, 1000)
+    ax.plot(support, rmodel.model.cdf(support), 'r-', label=modelname)
+    ax.legend()
+    return fig
+
+
+def SaveToPDF(fig, filename):
```

Comment:
Do we only want to allow PDF? It may also be good to let the user save as PNG etc. Perhaps something to consider for later.